compiler/qsc_frontend/src/resolve.rs

```diff
@@ -182,6 +182,12 @@ pub(super) enum Error {
         #[label]
         span: Span,
     },
+    #[error("this namespace clobbers (replaces) an existing external namespace of the same name")]
```

Comment:
nitpick: I know "clobbered" is the right and even common term of art for this, but it feels odd in an error message for some reason. Perhaps "overwrites" would be similarly descriptive?